src/Sylius/Bundle/TaxonomyBundle/Doctrine/ORM/TaxonRepository.php

```diff
@@ -110,4 +111,25 @@ public function createListQueryBuilder()
     {
         return $this->createQueryBuilder('o')->leftJoin('o.translations', 'translation');
     }
+
+    /**
+     * @param QueryBuilder $queryBuilder
+     * @param null|string $locale
+     *
+     * @return QueryBuilder
+     */
+    private function createTranslationQueryBuilderPart(QueryBuilder $queryBuilder, $locale = null)
```

Comment:
To be correct it does not create translation query builder part, but it configures that part on top of existing query builder.